pkgs/by-name/nm/nmgui/package.nix

```
@@ -0,0 +1,119 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  python3,
```

```suggestion
  python3Packages,
```

iirc `python3Packages` is preferred over `python3.pkgs` ? I might be misremembering, so tell me if there is something saying differently, but iirc there were a couple subtle details with `python3.pkgs` in regards to splicing and how packages would behave on cross.

---

@Sigmanificient found the issue documenting this: https://github.com/NixOS/nixpkgs/issues/211340
However, there is a bunch of complicated stuff in there, its probably not a super good resource for new contributors...

The TL;DR as far as i understood it: we need splicing because build-system vs dependencies can be different platforms. Adding python3.pkgs to those breaks on cross, because they don't get spliced.